pkg/exchange/bybit/exchange.go

```diff
@@ -0,0 +1,44 @@
+package bybit
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/exchange/bybit/bybitapi"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// BIT is the platform currency of Bybit, pre-allocate static string here
+const BIT = "BIT"
+
+var log = logrus.WithFields(logrus.Fields{
+	"exchange": "bybit",
+})
+
+type Exchange struct {
+	key, secret string
+	client      *bybitapi.RestClient
+}
+
+func New(key, secret string) *Exchange {
+	client := bybitapi.NewClient()
+
+	// for public access mode
+	if len(key) > 0 && len(secret) > 0 {
+		client.Auth(key, secret)
+	}
+
+	return &Exchange{
+		key: key,
+		// pragma: allowlist nextline secret
+		secret: secret,
+		client: client,
+	}
+}
+
+func (e *Exchange) Name() types.ExchangeName {
+	return types.ExchangeBybit
+}
+
+func (e *Exchange) PlatformFeeCurrency() string {
+	return BIT
```

Comment:
Does Bybit support paying trading fees with "BIT"?

Reply:
oh no, the bit only support on launch pad...